### PR TITLE
feat(CO-48) Dungeon frame

### PR DESCRIPTION
**Pull request by matyaspeterfi**

### CODE REVIEW

File: src/app/components/dungeon/dungeon.component.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  private _events: string[];
+
+  constructor(private dungeonService: DungeonService, private _route: ActivatedRoute) {
+    this._events = [
+      'Rivers and seas boiling.',
+      'Forty years of darkness.',
+      'Earthquakes', 'Volcanoes',
+      'Human sacrifice',
+      'dogs and cats living together – mass hysteria'
+    ];
```
Comment (Galicz555) [inline src/app/components/dungeon/dungeon.component.ts]: I know this is for mock but you should delete the content of the array. At least that was the case for me earlier.

File: src/app/components/hero-action/hero-action.component.html [hunk 2]
```diff
@@ -1,3 +1,2 @@
-<div>
```
Comment (Galicz555) [inline src/app/components/hero-action/hero-action.component.html]: If you are making a mock component next time please tell it so in the file name too

Comment (matyaspeterfi) [reply src/app/components/hero-action/hero-action.component.html]: came from master, only modified the contents

File: src/app/components/dungeon/dungeon.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  private _events: string[];
+
+  constructor(private dungeonService: DungeonService, private _route: ActivatedRoute) {
+    this._heroId = Number(this._route.snapshot.paramMap.get('id'));
+    this.dungeonService.getDungeon(this._heroId).subscribe(response => {
+      this._dungeon = response;
+    });
+  }
+
+  ngOnInit() {
```
Comment (juhaszPeterDev) [inline src/app/components/dungeon/dungeon.component.ts]: if youre not using oninit, dont include it

File: src/app/components/enemy-list-mock/enemy-list-mock.component.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  templateUrl: './enemy-list-mock.component.html',
+  styleUrls: ['./enemy-list-mock.component.scss']
+})
+export class EnemyListMockComponent implements OnInit {
+  _selectedObstacle: Obstacle;
+
+  @Input() dungeon: Dungeon;
+  @Output() _valueChange = new EventEmitter;
+
+  constructor() { }
```
Comment (juhaszPeterDev) [inline src/app/components/enemy-list-mock/enemy-list-mock.component.ts]: unnecessary

File: src/app/components/enemy-list-mock/enemy-list-mock.component.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+  ngOnInit() {
```
Comment (juhaszPeterDev) [inline src/app/components/enemy-list-mock/enemy-list-mock.component.ts]: unnecessary

File: src/app/components/dungeon/dungeon.component.ts (hunk 3 above)
Comment (matyaspeterfi) [reply src/app/components/dungeon/dungeon.component.ts]: fixed

File: src/app/components/enemy-list-mock/enemy-list-mock.component.ts (hunk 5 above)
Comment (matyaspeterfi) [reply src/app/components/enemy-list-mock/enemy-list-mock.component.ts]: fixed

File: src/app/components/enemy-list-mock/enemy-list-mock.component.ts (hunk 4 above)
Comment (matyaspeterfi) [reply src/app/components/enemy-list-mock/enemy-list-mock.component.ts]: fixed

**Pull request merged**